components/prism-ini.js

```diff
@@ -1,11 +1,37 @@
 Prism.languages.ini= {
-	'comment': /^[ \t]*[;#].*$/m,
-	'selector': /^[ \t]*\[.*?\]/m,
-	'constant': /^[ \t]*[^\s=]+?(?=[ \t]*=)/m,
-	'attr-value': {
-		pattern: /=.*/,
+	'comment': {
+		pattern: /(^[ \f\t\v]*)[#;][^\n\r]*/m,
+		lookbehind: true
+	},
+	'header': {
+		pattern: /(^[ \f\t\v]*)\[[^\n\r\]]*\]?/m,
```

Comment:
The last `\]` is optional because of the [`[foo` case](https://github.com/PrismJS/prism/issues/2775#issuecomment-787477723), right?

Reply:
Yes. This is how [`GetPrivateProfileSectionNames`](https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-getprivateprofilesectionnames) works, although this behavior is undocumented.